tasks/release.js

```diff
@@ -43,9 +43,8 @@ rm( 'dist/*.map' );
 // copy the directories to the release folder
 cp( '-Rf', filesToCopy, targetFolder );
 
-// copy the multi-currency files
-mkdir( '-p', targetFolder + '/multi-currency' );
-cp( '-R', 'multi-currency/src', targetFolder + '/multi-currency/src' );
+// remove the multi-currency/client source files
```

Comment:
I was wondering why previously we were copying from `multi-currency/src` to `release/woocommerce-payments/multi-currency/src` and now we are just deleting the `/includes/multi-currency/client`.

Should we add a note explaining why we removed this directory (e.g. `/includes/multi-currency/client`)? What do you think about adding the following?
```
Note: The '/includes/multi-currency/client' directory is removed because
`/includes/multi-currency/*` are server-side files. Furthermore, the `./client` directory is
already included in the `dist` directory during the `npm run build:client` step.
```

Reply:
> I was wondering why previously we were copying from multi-currency/src to release/woocommerce-payments/multi-currency/src and now we are just deleting the /includes/multi-currency/client.

This is because `includes` gets copied entirely [here](https://github.com/Automattic/woocommerce-payments/blob/bbe3b0e9d47ea5c6800cfbaa6254a74b57021535/tasks/release.js#L18), so instead of copying `multi-currency` from root and then deleting `multi-currency/client` we were simply copying `multi-currency/src`.

I agree this deserves better clarification, so I added a comment from your suggestion in https://github.com/Automattic/woocommerce-payments/pull/9681/commits/bbe3b0e9d47ea5c6800cfbaa6254a74b57021535.